query_test.go

```diff
@@ -550,6 +558,12 @@ func TestQueryEffectiveBufferSizeHeterogeneous(t *testing.T) {
 	err = arraySchema.AddAttributes(attribute)
 	assert.Nil(t, err)
 
+	err = arraySchema.AddAttributes(attribute2)
+	assert.Nil(t, err)
+
+	err = arraySchema.AddAttributes(attribute3)
```

Comment:
Does it need to be set as nullable?

Reply:
Yes both need to be set to nullable, and a2 need to be set to var length also.

Reply:
Done